salesforce-connector-demo/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.axonivy.salesforce.connector.demo</groupId>
+  <artifactId>salesforce-connector-demo</artifactId>
+  <version>1.0.0</version>
```

Comment:
Please define it as the same version of ivy version and add a postfix -SNAPSHOT, e.g: 10.0.0-SNAPSHOT, @HoDacVu90? Thanks!